addon-test-support/fire-event.js

```diff
@@ -42,14 +42,12 @@ export function fireEvent(element, type, options = {}) {
   @method buildBasicEvent
   @param {String} type
   @param {Object} (optional) options
-  @param {Boolean} (optional) bubbles
-  @param {Boolean} (optional) cancelable
   @return {Event}
   @private
 */
-function buildBasicEvent(type, options = {}, bubbles = true, cancelable = true) {
+function buildBasicEvent(type, options = {}) {
   let event = document.createEvent('Events');
-  event.initEvent(type, bubbles, cancelable);
+  event.initEvent(type, true, true);
```

Comment:
You could however extract `bubbles` and `cancelable` from the options object if that gives you a better API, but I think that you need to use the constructor.